manager-rs/dauth-service/src/rpc/handlers/local_authentication.rs

```diff
@@ -53,24 +56,34 @@ impl LocalAuthentication for LocalAuthenticationHandler {
     ) -> Result<tonic::Response<AkaConfirmResp>, tonic::Status> {
         tracing::info!("Request: {:?}", request);
 
-        let res_star = request.into_inner().res_star;
-        let res_star: auth_vector::types::ResStar =
-            res_star.try_into().or_else(|_e: Vec<u8>| {
-                Err(tonic::Status::new(
-                    tonic::Code::OutOfRange,
-                    "Unable to parse res_star",
-                ))
-            })?;
+        match LocalAuthenticationHandler::confirm_auth_hlp(
+            self.context.clone(),
```

Comment:
If the confirm_auth_hlp function is a method on LocalAuthenticationHandler, it can just access self.context directly and doesn't need to clone, right?